ll-time-input.html

```diff
@@ -84,10 +93,31 @@
      * @private
      */
     _isoToDisplay: function (isoStr) {
-      var m = moment(isoStr, 'HH:mm:ss');
-      if (m.isValid()) {
+      var m = this._getValidIsoMoment(isoStr);
```

Comment:
what's the advantage of this new function, `_getValidIsoMoment` ... it seems like it does the same thing as the code it's replacing but just adds a few extra lines and one extra step.